.github/workflows/v10-release.yml

```diff
@@ -36,10 +36,6 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Prepare artifacts for release
-        run: |
-          zip -r --junk-paths carbon-elements.sketchplugin.zip packages/sketch/carbon-elements.sketchplugin
-
```

Comment:
This is generated by the package sketch?, where this zip comes from?

@tay1orjones